configs/JHEF435/config.h

```diff
@@ -97,4 +97,5 @@
 #define MAX7456_SPI_INSTANCE    SPI2
 #define FLASH_SPI_INSTANCE      SPI3
 #define BARO_I2C_INSTANCE       (I2CDEV_2)
+#define MAG_I2C_INSTANCE        (I2CDEV_2)
```

Comment:
No idea why everyone puts braces around `I2CDEV_X`
```suggestion
#define MAG_I2C_INSTANCE        I2CDEV_2
```

Reply:
I don't think it makes a difference. We can clean this up and standardize in a future PR.